GPTPullRequestReview/src/review.ts

```diff
@@ -29,9 +27,17 @@ export async function reviewFile(targetBranch: string, fileName: string, httpsAg
 
     if (openai) {
       const response = await openai.createCompletion({
-        model: "text-davinci-003",
-        prompt: prompt,
-        max_tokens: 500
+        model: tl.getInput('model') || 'gpt-3.5-turbo',
+        messages: [
+          {
+            "role": "system",
+            "content": instructions
```

Comment:
the variable "instructions" is not declared in this file, anyway i will not suggest to make the prompt configurable because we can't control what the user can set as text for it.